meeteval/wer/__main__.py

```diff
@@ -525,6 +553,28 @@ def add_argument(self, command_parser, name, p):
                      '- lower,rm(.?!,): Lowercase the transcript and remove punctuations (.,?!).',
                 choices=[None, 'lower,rm(.?!,)'],
             )
+        elif name == 'distancetype':
```

Comment:
Should we really allow to change the value?
 - 1: It fails too often.
 - 2: It is unlikely to yield a desired alignment
 - 21: The only one, that we properly tested

Reply:
Agreed, I'll remove the argument